lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -10,259 +10,333 @@
-                     TopicsView, TopicModel, TopicCollection, TeamsView, TeamCollection, teamsTemplate) {
-               var ViewWithHeader = Backbone.View.extend({
-                   initialize: function (options) {
-                       this.header = options.header;
-                       this.main = options.main;
-                   },
+        function (Backbone, _, gettext, HeaderView, HeaderModel, TabbedView,
+                  TopicsView, TopicModel, TopicCollection, TeamProfileView, TeamsView, TeamCollection, TeamModel,
+                  teamsTemplate) {
+            var ViewWithHeader = Backbone.View.extend({
+                initialize: function (options) {
+                    this.header = options.header;
+                    this.main = options.main;
+                },
 
-                   render: function () {
-                       this.$el.html(_.template(teamsTemplate));
-                       this.$('p.error').hide();
-                       this.header.setElement(this.$('.teams-header')).render();
-                       this.main.setElement(this.$('.page-content')).render();
-                       return this;
-                   }
-               });
+                render: function () {
+                    this.$el.html(_.template(teamsTemplate));
+                    this.$('p.error').hide();
+                    this.header.setElement(this.$('.teams-header')).render();
+                    this.main.setElement(this.$('.page-content')).render();
+                    return this;
+                }
+            });
+            var TeamTabView = Backbone.View.extend({
+                initialize: function(options) {
+                    var TempTabView, router;
+                    this.courseID = options.courseID;
+                    this.topics = options.topics;
+                    this.topicUrl = options.topicUrl;
+                    this.teamsUrl = options.teamsUrl;
+                    this.maxTeamSize = options.maxTeamSize;
+                    // This slightly tedious approach is necessary
+                    // to use regular expressions within Backbone
+                    // routes, allowing us to capture which tab
+                    // name is being routed to
+                    router = this.router = new Backbone.Router();
+                    _.each([
+                        [':default', _.bind(this.routeNotFound, this)],
+                        ['topics/:topic_id', _.bind(this.browseTopic, this)],
+                        ['teams/:topic_id/:team_id', _.bind(this.browseTeam, this)],
+                        [new RegExp('^(browse)$'), _.bind(this.goToTab, this)],
+                        [new RegExp('^(teams)$'), _.bind(this.goToTab, this)]
+                    ], function (route) {
+                        router.route.apply(router, route);
+                    });
+                    // TODO replace this with actual views!
+                    TempTabView = Backbone.View.extend({
```

Comment:
Yes, this is providing the fake "My Teams" tab. It will be replaced when that tab is implemented.